docs/resize-single-disk-pool.md

```diff
@@ -0,0 +1,133 @@
+### 1. Do kubectl exec inside the `cstor-pool-mgmt` container using the following command and install `parted`
+
+Get the pool pod name using `kubectl get pods -n openebs` command and exec inside 
+the container. Install the `parted` tool using `apt-get install parted` after execing into the 
+`cstor-pool-mgmt` container.
+
+```
+$ kubectl exec -it cstor-pool-1fth-7fbbdfc747-sh25t -n openebs -c cstor-pool-mgmt bash
+```
+
+Install the `parted` tool using `apt-get install parted` after execing into the 
+`cstor-pool-mgmt` container.
+
+
+### 2. Run the following command inside the pool pod container:
+
+```
+zpool list (get the pool name)
+```
+**Example:**
+
+```
+$ zpool list
+NAME                                         SIZE  ALLOC   FREE  EXPANDSZ   FRAG    CAP  DEDUP  HEALTH  ALTROOT
+cstor-5be1d388-60d3-11e9-8e67-42010aa00fcf  9.94G   220K  9.94G         -     0%     0%  1.00x  ONLINE  -
+```
+
+### 3. Set your zpool with `autoexpand on` (By default it is set to `off`)
+
+```
+$ zpool set autoexpand=on <cstor-pool-name>
+```
+**Example:**
+```
+$ zpool set autoexpand=on cstor-5be1d388-60d3-11e9-8e67-42010aa00fcf
+```
+
+### 4. Resize the disk which is already part of the pool 
+
+If this is done already, that's fine.
+
+Now check `lsblk` output in corresponding node is showing the reflect size. General command for rescanning the SCSI bus on Linux is below.
+
+```
+sudo su -
+echo "1" > /sys/class/block/<device_name>/device/rescan
+```
+
+**Example:**
+
+```
+sudo su -
+echo "1" > /sys/class/block/sdb/device/rescan
+```
+
+**Note:**  This command can be different based on the underlying system. Please use the proper command to rescan the SCSI bus based on the underlying OS.
```

Comment:
Above command need to be run from the host machine (not from cstor-pool-mgmt container) on which disk has been resized.

Reply:
Done..